### ISSUE TITLE
unable to set an A record to the bare domain (i.e. t0m.us)

### ISSUE THREAD

**Issue by tomhoover** [barnybug/cli53#192]
### cli53 version (cli53 --version)

I installed using "brew install cli53"; however, "cli53 --version" only returns the following:

    cli53 version

"brew info cli53" returns:

    cli53: stable 0.8.0

### OS / Platform

OS X El Capitan

### Expected behavior

I've been using python/pypi cli53 to update my bare domain's (t0m.us) A record using the following command line:

    cli53 rrcreate --replace --ttl 300 t0m.us @ A 11.111.111.111

### Actual behaviour

After switching to the current cli53, I thought I'd be able to accomplish the same thing using the following command line:

    cli53 rrcreate --replace t0m.us '@ 300 A 11.111.111.111'

I also tried other variations (for example, without the TTL), such as:

    cli53 rrcreate --replace t0m.us '@ A 11.111.111.111'

and I just can't seem to find the proper command line to update the bare domain. Is this no longer possible with the current version? Or, am I missing something simple?

Any attempt to include the '@' sign to signify the bare domain, results in the following error message:

    Error: dns: not a TTL: "'@" at line: 2:2

**Comment by tomhoover** [comment]
During continued testing, I found a docker container (jeekajoo/alpine-cli53) that worked as I thought it should. Now that I knew that things should work, I decided the problem might be caused by having both the old and new versions of cli53 installed, I removed both the python, and go, versions of cli53. I then reinstalled just the go version via 'brew install cli53'. This seems to have corrected the issue.
